Subject: Fix call to update_prelight()

Seems like some minus signs sneaked in while diffing and reapplying...
Fixes regression noted in 480065.
---
diff --git a/gtk/gtktreeview.c b/gtk/gtktreeview.c
index 1a60677..949250f 100644
--- a/gtk/gtktreeview.c
+++ b/gtk/gtktreeview.c
@@ -2475,8 +2475,8 @@ gtk_tree_view_size_allocate (GtkWidget     *widget,
           if (tree_view->priv->prev_width_before_expander
               != width_before_expander)
               update_prelight (tree_view,
-                               -tree_view->priv->event_last_x,
-                               -tree_view->priv->event_last_y);
+                               tree_view->priv->event_last_x,
+                               tree_view->priv->event_last_y);
 
           tree_view->priv->prev_width_before_expander = width_before_expander;
         }


